idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -422,53 +414,48 @@ def _make_performance(self):
 
         # Add object references - others
         R_ref = CONST.gas_constant
+        g_ref = CONST.acceleration_gravity
 
         # Unit Model Parameters/sets
         self.zi = Param(self.vapor_phase.length_domain, mutable=True,
                         doc='''Integer indexing parameter required for transfer
                              across boundaries of a given volume element''')
-        # Set the integer  indices
+        # Set the integer indices along vapor phase length domain
         for i, x in enumerate(self.vapor_phase.length_domain, 1):
             self.zi[x] = i
 
         # Continuation parameters for initialization
-        self._homotopy_par_m = Param(initialize=0, mutable=True, units=None,
-                                     doc='''Continuation parameter to turn on mass
-                                     transfer terms gradually''')
-        self._homotopy_par_h = Param(initialize=0, mutable=True, units=None,
-                                     doc='''Continuation parameter to turn on heat
-                                     transfer terms gradually''')
-
-        # fixed column pressure
-        if self.config.fix_column_pressure:
-            self.column_pressure = Param(initialize=self.config.column_pressure,
-                                         mutable=True,
-                                         units=pyunits.Pa,
-                                         doc='Fixed operating pressure of column')
-
-        # Interfacial area  parameters
+        self._homotopy_par_m = Param(
```

Comment:
1. We should look at whether we actually need this, especially if we start with a simpler model with fixed mass transfer coefficients, enhancement factor and pressure drop.
2. I had some success in earlier testing of removing the energy balance homotopy step at least.
3. If it is needed, this should be an argument to initialization, not a parameter on the model.